src/lib/types.ts

```diff
@@ -110,6 +112,13 @@ interface ImportingUser {
   email: string;
 }
 
+export interface CommandResult {
```

Comment:
Can we unify this a bit more so we are not returning empty strings?
If there is nothing to return send undefined or don't send it at all.Checking for empty string is not very intuitive.

```
export type CommandResult {
  message: string;
  exitCode: number;
  filename?: string;
}

Reply:
`filename` => `fileName`